### PR TITLE
[CN] Added "Loading a global settings file"

### PR DESCRIPTION
**Pull request by Jinjiang**
ref #961

### CODE REVIEW

File: docs/zh-cn/configurations/pre-processors.md [hunk 1]
```diff
@@ -37,6 +37,40 @@ npm install sass-loader node-sass --save-dev
 
 如要获得更多关于 `vue-loader` 的配置信息，请查看 [Loader 进阶配置](./advanced.md) 章节。
 
+### 加载一个全局设置文件
+
+在每个组件里加载一个设置文件，而无需每次都将其显示导入，是一个常见的需求。比如为所有组件全局使用 scss 变量。为了达成此目的：
```
Comment (fnlctrl) [inline docs/zh-cn/configurations/pre-processors.md]: 一个小typo😂: 显示 => 显式

Comment (Jinjiang) [reply docs/zh-cn/configurations/pre-processors.md]: Done

**Comment by kazupon** [comment]
@Jinjiang @fnlctrl thanks!

**Pull request merged**